docs/manuals/core/entities/sequence-mgt.md

```diff
@@ -0,0 +1,111 @@
+- _**tasks**_: The dictionary holding the various tasks of the sequence. The key corresponds to the config_id of the
+  task while the value is the task itself.
+- _**data_nodes**_: The dictionary holding the various data nodes of the sequence. The key corresponds to the
+    data node's _config_id_ (while the value is the data node itself).
+- _**owner_id**_: The identifier of the owner, which can be a scenario, cycle, or None.
+- _**version**_: The string indicates the application version of the sequence to instantiate. If not provided,
+  the current version is used. Refer to the [version management](../versioning/index.md) page for more details.
+- Each property of the _**properties**_ dictionary is also directly exposed as an attribute.
+- Each nested entity is also exposed as an attribute of the sequence. The attribute name corresponds to the
+  *config_id* of the nested entity.
+
+!!! Example
+
+    ```python linenums="1"
+    import taipy as tp
+    from datetime import datetime
+    import my_config
+
+    scenario = tp.create_scenario(my_config.monthly_scenario_cfg, name="Monthly scenario")
+    sales_sequence = scenario.sales_sequence
+    sales_sequence.name = "Sequence for sales prediction"
+
+    # There was no subscription, so subscribers is an empty list
+    sequence.subscribers # []
+    # The properties dictionary equals {"name": "Sequence for sales prediction"}. It
+    # contains all the properties, including the `name` provided at the creation
+    sequence.properties # {"name": "Sequence for sales prediction"}
+    # The `name` property is also exposed directly as an attribute. It
+    # equals "Sequence for sales prediction"
+    sequence.name
+    # The training task entity is exposed as an attribute of the sequence
+    training_task = sequence.training
+    # The predicting task entity as well
+    predicting_task = sequence.predicting
+    # The data nodes are also exposed as attributes of the sequence.
+    current_month_data_node = sequence.current_month
+    ```
+
+# Get a sequence
+
+There are two ways to retrieve a sequence:
+
+The first method is by getting it from its parent scenario, using the sequence name as an attribute.
+The second method to access a sequence is from its id by using the `taipy.get()^` method.
+
+```python linenums="1"
+import taipy as tp
+import my_config
+
+scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
+sequence = scenario.sales_sequence
+
+sequence_retrieved = tp.get(sequence.id)
+sequence == sequence_retrieved
+```
+
+Here the two variables `sequence` and `sequence_retrieved` are equal.
+
+# Get all sequences
+
+All the sequences can be retrieved using the method `taipy.get_sequences()^`. This method returns the list of all
+existing sequences.
+
+# Delete a sequence
+
+A sequence can be deleted by using `taipy.delete()^` which takes the sequence id as a parameter. The deletion is
```

Comment:
```suggestion
A sequence can be deleted using `taipy.delete()^` which takes the sequence id as a parameter. The deletion is
```

Reply:
I would remove the "which takes the sequence id as a parameter". Useless.